cmd/tke-installer/app/installer/installer.go

```diff
@@ -1517,6 +1517,14 @@ func (t *TKE) prepareBaremetalProviderConfig(ctx context.Context) error {
 			Name: "docker",
 			File: baremetalconstants.ConfDir + "docker/*",
 		},
+		{
+			Name: "containerd",
+			File: baremetalconstants.ConfDir + "containerd/*",
+		},
+		{
+			Name: "critools",
```

Comment:
Why we need critools config? When we create busniss cluster, any works need critools?